src/CommunityToolkit.Maui/Controls/Avatar.cs

```diff
@@ -0,0 +1,557 @@
+using System.ComponentModel;
+using Microsoft.Maui.Controls.Internals;
+using Microsoft.Maui.Controls.Shapes;
+
+namespace CommunityToolkit.Maui.Controls;
```

Comment:
Please move this file to `/Views/` folder

Reply:
I have moded this file to the /Views/ folder and renamed to AvatarView.cs. I believe I have resolved this issue.